Clinical-Genomics/fluffy
# FLuffy fail failed sucessfully

### This PR adds | fixes:
- yaml file listing all submitted job ids
- renamed the gc picard output file
- added the multiqc report to deliverables file
- removed the qc zip files from the deliverables file

**How to prepare for test**:
- [ ] `ssh` to ...
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
- install fluffy
- run fluffy
- check the output csv, analysis_status and multiqc files

### Expected outcome:
- [ ] analysis status complete
- [ ] fluffy should produce a yaml file listing all jobs

### Review:

This [version](https://semver.org/) is a:
